colab/Colab_UnityEnvironment_1_Run.ipynb

```diff
@@ -32,7 +32,7 @@
    },
    "source": [
     "# ML-Agents Open a UnityEnvironment\n",
-    "<img src=\"https://github.com/Unity-Technologies/ml-agents/blob/release_1/docs/images/image-banner.png?raw=true\" align=\"middle\" width=\"435\"/>"
+    "<img src=\"https://github.com/Unity-Technologies/ml-agents/blob/release_18/docs/images/image-banner.png?raw=true\" align=\"middle\" width=\"435\"/>"
```

Comment:
TODO make sure this image links still work. May need to move the images to a more permanent location.

Reply:
Broken, will fix.